### PR TITLE
LPS-200779 | Test Fix | master

### PR DESCRIPTION
**Pull request by kristinonias**
[LPS-200779](https://liferay.atlassian.net/browse/LPS-200779)

cc: @patriciajeanperez

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase [hunk 1]
```diff
@@ -373,6 +373,10 @@ definition {
 	test CanOrderByCreatedBy {
 		var siteName = TestCase.getSiteName(siteName = ${siteName});
 
+		Site.openSiteSettingsAdmin(siteURLKey = "guest");
```
Comment (kristinonias) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:376]: Added these 2 macros to tests that include membership. Membership needs to be enabled through site settings for it to be visible in the side panel.

Comment (timpak) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:376]: The change looks fine to me but I'll let Patty take a look once she's back

**Comment by patriciajeanperez** [comment]
ci:forward

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**